wire-ios-transport/WireTransportSupport/HTTPCookie+Testing.swift

```diff
@@ -16,14 +16,16 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
+@testable import WireTransport
 
-@objc public protocol AuthenticationStatusProvider {
```

Comment:
I've no idea why git sees this as a move 🤔 The before and after are 2 unrelated files.

Anyway, deleted the `AuthenticationStatusProvider` **protocol** because it is not used. There is an unrelated class, `AuthenticationStatusProvider` which is still in use.